admin-legacy/uagb-admin-loader.php

```diff
@@ -0,0 +1,590 @@
+<?php
+/**
+ * UAGB Admin.
+ *
+ * @package UAGB
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! class_exists( 'UAGB_Old_Admin' ) ) {
+
+	/**
+	 * Class UAGB_Old_Admin.
+	 */
+	final class UAGB_Old_Admin {
```

Comment:
```suggestion
	final class UAGB_Legacy_Admin {
```

CHange file name accordingly